subgraphs/apeswap/schema.graphql

```diff
@@ -0,0 +1,557 @@
+# Subgraph Schema: DEX AMM
+# Version: 1.0.1
+
+enum Network {
+  ARBITRUM
+  AURORA
+  AVALANCHE
+  BSC
+  CELO
+  CRONOS
+  ETHEREUM
+  FANTOM
+  HARMONY
+  MOONBEAM
+  MOONRIVER
+  OPTIMISM
+  POLYGON
+  XDAI
+}
+
+enum ProtocolType {
+  EXCHANGE
+  LENDING
+  YIELD
+  BRIDGE
+  GENERIC
+  # Will add more
+}
+
+type Token @entity {
+  " Smart contract address of the market "
+  id: ID!
+
+  " Mirrored from the smart contract "
+  name: String!
+  symbol: String!
+  decimals: Int!
+
+  ###### internal use ########
+  # used for other stats like marketcap
+  totalSupply: BigDecimal!
```

Comment:
prefix with underscore for internal fields

Reply:
Fixed